zipkin-lens/src/components/Timeline/index.js

```diff
@@ -135,6 +135,7 @@ class Timeline extends React.Component {
                   startTs={startTs}
                   endTs={endTs}
                   traceDuration={traceSummary.duration}
+                  traceTimestamp={traceSummary.spans[0].timestamp}
```

Comment:
is this different than traceSummary.timestamp?

Reply:
`traceSummary` does not have `timestamp` field...

Reply:
heh ok!